src/main/java/mate/academy/bookshop/controller/BookController.java

```diff
@@ -28,6 +31,12 @@ public BookDto getBookById(@PathVariable Long id) {
         return bookService.findById(id);
     }
 
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @DeleteMapping("/{id}")

```

Comment:
You are missing a PUT endpoint for existing book update & a service method for it, check the task description once again